### PR TITLE
Splits high and low pressure, nerfs emergency firesuits

### PR DESCRIPTION
**Pull request by Moltijoe**
Requested by @Aquizit sorta

# Why is this good for the game?
Makes space walking a bit more special than getting internals, drinking some coffee, and slapping on a firesuit

# Testing
low
high

:cl:  
tweak: Emergency firesuits no longer protect against low pressure
/:cl:

### CODE REVIEW

**Comment by tattax** [comment]
this is a plasmaman nerf btw

**Comment by Scrambledeggs00** [comment]
Finally, a plasmeme nerf for extra challenge ;]

**Comment by ShoziOffical** [comment]
so can you still survive in a spaced room with firesuit?

**Comment by Moltijoe** [comment]
> so can you still survive in a spaced room with firesuit?

no, that's the point
unless you mean atmos firesuit, that one's still able to

**Comment by ShoziOffical** [comment]
> > so can you still survive in a spaced room with firesuit?
> 
> no, that's the point unless you mean atmos firesuit, that one's still able to

Aigh cool

File: code/modules/mob/living/carbon/human/life.dm [hunk 1]
```diff
@@ -74,15 +74,20 @@
...
+			if(CH && istype(CH))
+				if(pressure > ONE_ATMOSPHERE && (CS.clothing_flags & CH.clothing_flags & STOPSHIGHPRESSURE))
+					return ONE_ATMOSPHERE
+				else if(pressure < ONE_ATMOSPHERE && (CS.clothing_flags & CH.clothing_flags & STOPSLOWPRESSURE))
+					return ONE_ATMOSPHERE
+		else // you don't need a helmet if you don't have a head
+			if(pressure > ONE_ATMOSPHERE && (CS.clothing_flags & STOPSHIGHPRESSURE))
+				return ONE_ATMOSPHERE
+			else if(pressure < ONE_ATMOSPHERE && (CS.clothing_flags & STOPSLOWPRESSURE))
+				return ONE_ATMOSPHERE
```
Comment (tattax) [inline code/modules/mob/living/carbon/human/life.dm:90]: mfw else if after return...

Comment (Moltijoe) [reply code/modules/mob/living/carbon/human/life.dm:90]: realistically, we shouldn't be having several return statements if we were following best coding practices, but training is fighting with shitcode

**Comment by MajManatee** [comment]
makes sense

**Review by ToasterBiome — APPROVED** [review]
space should be dangerous, sowwy

**Pull request merged**